unison-src/transcripts/doc-formatting.md

```diff
@@ -20,10 +20,10 @@ scratch/main> add
 scratch/main> view foo
 ```
 
-Note that `@` and `:]` must be escaped within docs.
+Note that `{{`, `@`, and `}}` must be escaped within docs.
 
 ```unison
-escaping = [: Docs look [: like \@this \:] :]
+escaping = {{ Docs look --doc open-- like --amphora--this --doc close-- }}
```

Comment:
I wasn’t sure how to convert this bit – AFAICT, there’s no support for escaping characters in Doc v2?